packages/expo-video-thumbnails/android/src/main/java/expo/modules/videothumbnails/Exceptions.kt

```diff
@@ -0,0 +1,12 @@
+package expo.modules.videothumbnails
+
+import expo.modules.kotlin.exception.CodedException
+
+class ThumbnailException :
```

Comment:
```suggestion
class ThumbnailFileException :
```
Let's rename that to something more meaningful